nethermind/quickstart.sh

```diff
@@ -0,0 +1,652 @@
+        CONFIG="fuse"
+
+        NETSTATS_DOCKER_IMAGE=$NETSTATS_CLIENT_DOCKER_IMAGE
+        NETSTATS_VERSION=$NETSTATS_CLIENT_DOCKER_IMAGE_VERSION
+    fi
+
+    # For explorer (archive)
+    if [[ $NETWORK == "fuse" && $ROLE == "explorer" ]]; then
+        CONTAINER_NAME="fuse"
+        DB_PREFIX="fuse_archive"
+        CONFIG="fuse_archive"
+
+        NETSTATS_DOCKER_IMAGE=$NETSTATS_CLIENT_DOCKER_IMAGE
+        NETSTATS_VERSION=$NETSTATS_CLIENT_DOCKER_IMAGE_VERSION
+    fi
+
+    # For validator
+    if [[ $NETWORK == "fuse" && $ROLE == "validator" ]]; then
+        CONTAINER_NAME="fuse"
+        DB_PREFIX="fuse_validator"
+        CONFIG="fuse_validator"
+
+        VALIDATOR_DOCKER_IMAGE=$FUSE_VALIDATOR_DOCKER_IMAGE
+        NETSTATS_DOCKER_IMAGE=$SPARK_NETSTATS_CLIENT_DOCKER_IMAGE
+        NETSTATS_VERSION=$SPARK_NETSTATS_CLIENT_DOCKER_IMAGE_VERSION
+    fi
+
+    # Run Docker container
+    if [[ $ROLE == "node" || $ROLE == "bootnode" || $ROLE == "explorer" ]]; then
+        $PERMISSION_PREFIX docker run \
+            --detach \
+            --name $CONTAINER_NAME \
+            --volume $DATABASE_DIR:/nethermind/nethermind_db/$DB_PREFIX \
+            --volume $KEYSTORE_DIR:/nethermind/keystore \
+            --volume $LOGS_DIR:/nethermind/logs \
+            --log-opt max-size=10m \
+            --log-opt max-file=25 \
+            --log-opt compress=true \
+            -p 30303:30300/tcp \
+            -p 30303:30300/udp \
+            -p 8545:8545 \
+            -p 8546:8546 \
+            --restart always \
+            $FUSE_CLIENT_DOCKER_IMAGE \
+            --config $CONFIG \
+            --Init.WebSocketsEnabled true \
+            --HealthChecks.Enabled true \
+            --HealthChecks.Slug /api/health
+
+        # Run Netstat
+        $PERMISSION_PREFIX docker run \
+            --detach \
+            --name "netstats" \
+            --net container:$CONTAINER_NAME \
+            --log-opt max-size=10m \
+            --log-opt max-file=25 \
+            --log-opt compress=true \
+            --restart always \
+            --memory "250m" \
+            $NETSTATS_DOCKER_IMAGE \
+            --instance-name $NODE_KEY \
+            --role ${ROLE^} \
+            --netstats-version $NETSTATS_VERSION
+    fi
+
+    if [[ $ROLE == "validator" ]]; then
+        $PERMISSION_PREFIX docker run \
+            --detach \
+            --name $CONTAINER_NAME \
+            --volume $DATABASE_DIR:/nethermind/nethermind_db/$DB_PREFIX \
+            --volume $KEYSTORE_DIR:/nethermind/keystore \
+            --volume $LOGS_DIR:/nethermind/logs \
+            --log-opt max-size=10m \
+            --log-opt max-file=25 \
+            --log-opt compress=true \
+            -p 30303:30300/tcp \
+            -p 30303:30300/udp \
+            -p 8545:8545 \
+            -p 8546:8546 \
+            --restart always \
+            $FUSE_CLIENT_DOCKER_IMAGE \
+            --config $CONFIG \
+            --JsonRpc.Host 0.0.0.0 \
```

Comment:
do we want to expose RPC for validators globally? shouldn't this just be 127.0.0.1 (for netstats only). Having this setting here will also require --JsonRpc.Enabled true to be set (as it will override this back to false I believe)

Reply:
Here I could disagree because of next things:

1. As you know we're running our nodes in AWS. To restrict an access to specific ports we use Security groups. For validators / bridges it could be restrict for specific whitelist IP's such as VPN, personal static addresses, etc.
2. If we want to monitor our validators we need an access to RPC endpoint - we collect such metrics to CloudWatch via Python script which do some RPC POST requests. If we're running JSON-RPC in Docker on 127.0.0.1 it means that we have an access to RPC only from Docker network.

Reply:
I think maybe we have this as an optional flag to keep this internal/external?. As currently we are exposing all RPC modules including admin and personal!. 

So we should also pass in EnabledModules Eth,Web3,Health,Rpc to ensure we are only enabling modules we need

Reply:
Okay, got you! Make sense!

Maybe let's specify here Docker command for each role. @Andrew-Pohl